### PR TITLE
More CSS stuff

### PR DESCRIPTION
**Pull request by fauxpark**
I couldn't stop myself

### CODE REVIEW

File: src/components/ErrorReport.vue [hunk 1]
```diff
@@ -161,6 +162,10 @@ export default {
   overflow: hidden;
   white-space: nowrap;
   border-radius: 8px;
+  display: flex;
```
Comment (yanfali) [inline src/components/ErrorReport.vue]: I'm confused about this bit. Why are you setting it? It actually changes the internal formatting of the div

Comment (fauxpark) [reply src/components/ErrorReport.vue]: image

**Comment by yanfali** [comment]
Try the following. Wrap the two text blocks in divs and watch what happens

**Comment by fauxpark** [comment]
I see it.

`display: grid;` seems to fix it.

**Comment by yanfali** [comment]
Works for me.

**Comment by yanfali** [comment]
You could also try flex-direction: column

**Comment by fauxpark** [comment]
Nope, no vertical centering with that.

**Pull request merged**